cli/src/utils/getdeps.ts

```diff
@@ -16,20 +16,20 @@ export async function getDependencies(projectDir: string) {
   const { dependencies: lockDeps } = JSON.parse(await readFile(packageLockPath, 'utf8'));
   const { dependencies: pkgDeps } = JSON.parse(await readFile(packageJsonPath, 'utf8'));
 
-  const toProcess = Object.keys(pkgDeps);
+  const toProcess = Object.keys(pkgDeps).map((dep) => [dep, lockDeps]);
```

Comment:
I think `root = lockDeps` is confusing - perhaps better to insert an empty object like `{}`. Also, this seems to break with npm dependencies https://www.npmjs.com/package/dependencies.